builtin/game/item.lua

```diff
@@ -156,7 +156,7 @@ function core.yaw_to_dir(yaw)
 end
 
 function core.get_node_drops(nodename, toolname)
-	local drop = ItemStack({name=nodename}):get_definition().drop
+	local drop = core.registered_nodes[nodename].drop
```

Comment:
This would throw an error for any unknown node.
`get_definition` returns the table of athe node `unknown`, thus this needs a crash-safe check.